opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -36,6 +36,16 @@ private Tags() {
     public static final String SPAN_KIND_CLIENT = "client";
 
     /**
+     * A constant for setting the span kind to indicate that it represents a producer span.
```

Comment:
let's mention that this is specifically relevant in the `message_bus` context... since client/server are so much more common.